tests/test_google_sheets_automatic_fields.py

```diff
@@ -64,3 +70,8 @@ def test_run(self):
                     expected_keys.remove('modifiedTime')
                 for actual_keys in record_messages_keys:
                     self.assertSetEqual(expected_keys, actual_keys)
+
+                #Verify that all replicated records have unique primary key values.
+                self.assertEqual(len(primary_keys_list), 
```

Comment:
I think this is actually covered by the pagination test since the pk is an `_sdc` field in all cases for this tap.

Reply:
Removed it.